docs/source/data-publishing/ogcapi-edr.rst

```diff
@@ -58,12 +58,29 @@ The `xarray-edr`_ provider plugin reads and extracts `NetCDF`_ and `Zarr`_ data
          format:
             name: zarr
             mimetype: application/zip
+    
+    providers:
+       - type: edr
+         name: xarray-edr
+         data: s3://power-analysis-ready-datastore/power_901_annual_meteorology_utc.zarr
+         format:
+            name: zarr
+            mimetype: application/zip
+        storage_options:
+            anon: true
+            requester_pays: false
 
 .. note::
 
    `Zarr`_ files are directories with files and subdirectories.  Therefore
    a zip file is returned upon request for said format.
 
+.. note::
+   When referencing data stored in an S3 bucket, be sure to provide the full
+   S3 URL. Any parameters required to open the dataset using fsspec can be added
+   to the config file under `storage_options`.
+
+
```

Comment:
Because this PR adds S3 support for both OA-EDR and  OA-Coverages, it might be worth adding this documentation to [ogcapi-coverages/xarray](https://github.com/geopython/pygeoapi/blob/master/docs/source/data-publishing/ogcapi-coverages.rst) as well

Reply:
Looks like @sjordan29 did that with https://github.com/geopython/pygeoapi/pull/1235/commits/ab198109b0f161a785874ca0b029776d0bda1ef8